roles/openshift_hosted/defaults/main.yml

```diff
@@ -26,6 +26,7 @@ openshift_cluster_domain: 'cluster.local'
 ##########
 r_openshift_hosted_router_firewall_enabled: "{{ os_firewall_enabled | default(True) }}"
 r_openshift_hosted_router_use_firewalld: "{{ os_firewall_use_firewalld | default(False) }}"
+r_openshift_hosted_router_prometheus_enabled: "{{ openshift_prometheus_state == 'present' | default(True) }}"
```

Comment:
note to reviewer, this may need `openshift_prometheus_state and openshift_prometheus_state == 'present'` but wasn't sure

Reply:
fixed